tests/unit/common.py

```diff
@@ -81,6 +81,11 @@ def set_accelerator_visible():
                 match = re.search('Device Type.*GPU', line)
                 if match:
                     num_accelerators += 1
+        elif get_accelerator().device_name() == 'hpu':
```

Comment:
FYI @vshekhawat-hlab - I stood up a runner in #5244 so we can test these types of changes.

Reply:
FYI @nelyahu as well

Reply:
thanks @loadams, looks promising.
@vshekhawat-hlab will be integrating with you.
keep me in the loop :)

Reply:
Sounds good, merged that PR, it contained the changes from util.py FYI @vshekhawat-hlab.

Reply:
@loadams ,

In order to setup Gaudi2 CI. I have raised the following PRs for the Gaudi2 accelerator:

#5162 
#5269 
#5270 
#5271 

Can you please review them.

Reply:
Yes, thanks @vshekhawat-hlab!